client/my-sites/sharing/connections/service.jsx

```diff
@@ -47,6 +47,15 @@ import requestExternalAccess from 'lib/sharing';
 import MailchimpSettings, { renderMailchimpLogo } from './mailchimp-settings';
 import config from 'config';
 
+/**
+ * Check if the connection is broken or requires reauth.
+ *
+ * @param {object} connection Publicize connection.
+ * @returns {boolean} True if connection is broken or requires reauthentication.
+ */
+const isConnectionInvalidOrReauth = connection =>
+	[ 'must_reauth', 'invalid' ].includes( connection.status );
```

Comment:
Do we have a polyfill for `.includes()` already? If not, we might want to use `lodash.includes()` instead.

Reply:
Calypso now includes [`babel-polyfill`](
https://babeljs.io/docs/en/babel-polyfill) so we're good:

https://github.com/Automattic/wp-calypso/blob/63917e91dd65dc95273759a786cb706b05d9f89e/index.js#L3